app/code/Magento/Integration/Test/Mftf/Test/AdminCreateIntegrationEntityWithDuplicatedNameTest.xml

```diff
@@ -46,5 +46,6 @@
             <argument name="message" value="The integration with name &quot;Integration1&quot; exists."/>
             <argument value="error" name="messageType"/>
         </actionGroup>
+        <seeInField stepKey="checkEnteredValueIsPreserved" selector="{{AdminNewIntegrationSection.name}}" userInput="Integration1"/>
```

Comment:
Hi @realadityayadav. According to the [best practices](https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices) it's highly recommended to build a test using action groups without direct operations in the test body. As you can see from the test you have edited, it consists of the action groups.

Please, move this check into a new action group and name the action group as `AssertAdminIntegrationNameInFormActionGroup`.  Then you can use the action group in your test.

Thank you!

Reply:
https://github.com/magento/magento2/pull/26660/files

Please see the recent change now @rogyar